src/attributes.md

```diff
@@ -21,11 +21,11 @@
 > &nbsp;&nbsp; | _MetaItem_  
 > &nbsp;&nbsp; | _MetaItem_ `,` _MetaSeq_
 
-Any item declaration may have an _attribute_ applied to it. Attributes in Rust
-are modeled on Attributes in ECMA-335, with the syntax coming from ECMA-334
-(C#). An attribute is a general, free-form metadatum that is interpreted
-according to name, convention, and language and compiler version. Attributes
-may appear as any of:
+Any [item declaration] or [generic lifetime or type parameter][generics] may
```

Comment:
or statement

Reply:
Or block expression

Reply:
Actually, this is a lot more complicated, because also enum variants, struct and union fields, match arms...